Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -1030,10 +1082,22 @@ void FiniteDifferenceSolver::HybridPICSolveECartesian (
                     Ex(i, j, k) += eta(rho_val, jtot_val) * Jx(i, j, k);
 
                     if (include_hyper_resistivity_term) {
+
+                        // Interpolate B field to appropriate staggering to match E field - mtobin 20240217
+                        Real btot_val = 0._rt;
+                        if (solve_for_Faraday && hyper_resistivity_has_B_dependence) {
```

Comment:
These could just be:
```suggestion
                        if (hyper_resistivity_has_B_dependence) {
```
since this whole block of code is inside a `if (solve_for_Faraday)` condition.